src/content/packages-and-plugins/swift-package-manager/for-plugin-authors.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Swift Package Manager for plugin authors
+description: How to add Swift Package Manager compatibility to iOS and macOS plugins
+diff2html: true
+---
+
+Flutter is migrating to Swift Package Manager to manage iOS and macOS native
+dependencies.
+This is an experimental feature that may change in the future.
+It is currently only available on the [`master` channel][].
+Flutter will continue to support CocoaPods until further notice.
+
+Swift Package Manager adoption will be gradual.
+
+Flutter recommends that Flutter plugins support _both_ Swift Package Manager
+and CocoaPods until further notice.
```

Comment:
if that's not easy to do, should we link to guidance for how to support both?

Reply:
This guidance does make your plugin support both package managers. I'll add a blurb to clarify this.

Reply:
I tweaked this intro, let me know if it's clearer now.